### PR TITLE
Mbl 81/socket redis testing

### PR DESCRIPTION
**Pull request by richchurcher**

### CODE REVIEW

**Comment by richchurcher** [comment]
Investigating the redis thing on Circle.

**Comment by richchurcher** [comment]
Ok @levity I think we're all working now...

File: circle.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 machine:
+  environment:
+    REDIS_URL: redis://0.0.0.0:6379/1
   node:
     version: 8.6.0
+  services:
+    - redis
```
Comment (levity) [inline circle.yml]: well isn't that pleasingly simple.

Comment (richchurcher) [reply circle.yml]: I know! If we didn't require a host in the Node source, I doubt I would even have had to provide an environment variable.

**Pull request merged**